src/main/kotlin/org/rust/ide/spellchecker/StringLiteralTokenizer.kt

```diff
@@ -22,9 +22,8 @@ class StringLiteralTokenizer : EscapeSequenceTokenizer<LeafPsiElement>() {
         fun processTextWithEscapeSequences(element: LeafPsiElement, text: String, consumer: TokenConsumer) {
             val unescapedText = StringBuilder()
             val offsets = IntArray(text.length + 1)
-            CodeInsightUtilCore.parseStringCharacters(text, unescapedText, offsets)
-
-            processTextWithOffsets(element, consumer, unescapedText, offsets, 1)
```

Comment:
This `1` was a bug, it caused the spellchecker note to be shifted one character to the right.

For `Hello, W\u{6F}dlr!`, the matched word was `Wodlr` (correctly), but actually `odlr!` was underlined.